src/sentry/static/sentry/app/utils.tsx

```diff
@@ -61,21 +62,13 @@ function objectMatchesSubset(obj, other, deep) {
   return true;
 }
 
-// XXX(dcramer): the previous mechanism of using _.map here failed
-// miserably if a param was named 'length'
-export function objectToArray(obj) {
-  const result = [];
-  for (const key in obj) {
-    result.push([key, obj[key]]);
-  }
-  return result;
-}
+export const objectToArray = Object.entries;
```

Comment:
I wonder if this will have implications, since right now this would iterate keys like `constructor`. Not sure if `Object.entries` will do that as well.

Reply:
I don't think we ever want to iterate keys like `constructor`, `Object.entries` only iterates its own properties.

Reply:
Good points. I'll revisit this to see if the refactor is generally small enough to fit into this PR.

I don't want to accidentally introduce a bug by "correcting" this utility function.

Reply:
Let's not tackle it here, IMO we should fix it though, I recall a case of a project named "constructor" recently that did have some problems.